### ISSUE TITLE
client secret过期，自建后报错消失

### ISSUE THREAD

**Issue by LongStudy** [spencerwooo/onedrive-vercel-index#1049]
今天打开网站变404了，好像是client secret过期了

**Comment by sudoalx** [comment]
Yes, the default client expired. I guess we'll have to use our own for now :crying_cat_face:

**Comment by wdw1007** [comment]
草了好像还真的是所有者的 client secret 过期了
本项目的第一次 commit [03c05b4](https://github.com/spencerwooo/onedrive-vercel-index/commit/03c05b4d662b224cb655ff33ee709ef94095d40c) 是在 2021/06/22
微软允许的最长时间刚好是两年 😂😂
![](https://github.com/spencerwooo/onedrive-vercel-index/assets/87262434/9032f3a7-f202-4729-92b9-84308e90585b)

**Comment by iRedScarf** [comment]
使用自己的重新部署，需要把Redis数据库重置，最简单的方法是删掉原来的Redis数据库再重建一个进行关联。

**Comment by LongStudy** [comment]
使用upstash部署的直接删了token以后换自己的client id和secret即可

**Comment by sudoalx** [comment]
> To use your own redeployment, you need to reset the Redis database. The easiest way is to delete the original Redis database and rebuild it for association.

Alternatively you can go to the setup route and the token will be overwritten in step 3
Route for oauth setup:
```
/onedrive-vercel-index-oauth/step-1/
```

**Comment by crushedoreos** [comment]
> Yes, the default client expired. I guess we'll have to use our own for now 😿

How do I access and use that?
